spec/requests/medications_spec.rb

```diff
@@ -236,15 +236,15 @@ def put_update(medication_params)
       end
 
       context 'when medication has a daily reminder' do
-        let!(:medication) do
-          create(:medication, :with_daily_reminder, user: user)
-        end
+        let!(:medication) { create(:medication, user: user) }
```

Comment:
There is an after creating a hook into the factories that creates a `take_medication_reminder` and a `refill_reminder`, as we have a `has_one` relation during the tests we have been creating two objects (2 take_medication_reminder and 2 refill_reminder) for the same medication, in other words, we are creating an inconsistency in the tests. So I fixed that here.